src/test/app/PayStrand_test.cpp

```diff
@@ -857,6 +844,30 @@ struct PayStrand_test : public beast::unit_test::suite
             }
         }
     }
+    void
+    run() override
+    {
+        testAllPairs({featureFlow, fix1373});
+        testAllPairs({featureFlow, fix1373, featureFlowCross});
+    }
+};
+
+BEAST_DEFINE_TESTSUITE_MANUAL(PayStrandAllPairs, app, ripple);
+
+struct PayStrand_test : public beast::unit_test::suite
+{
+    static bool hasFeature(uint256 const& feat)
+    {
+        return false;
+    }
+
+    static bool hasFeature(uint256 const& feat, std::initializer_list<uint256> args)
+    {
+        for(auto const& f : args)
```

Comment:
I apparently failed to click `Add` on this comment...  For fans of Sean ("no naked for loops") Parent, here's an alternative to this `for` loop:
```
    return (std::find (args.begin(), args.end(), feat) != args.end());
```
No change needed...